src/components/ApplicationsModal/ApplicationsModal.tsx

```diff
@@ -47,29 +47,47 @@ export const ApplicationsModal: React.FC<ApplicationsModalProps> = ({
   const { t } = useTranslation();
   const { navigation, ownerInfo, portalName } = useConfig();
   const dispatch = useDispatch();
-  const history = useHistory();
+  const history: any = useHistory();
```

Comment:
Avoid using any.

Reply:
Even when setting a type for the `history` variable, TS errors occurred (either related to the `location`'s passed-in `state`, or the fact that history has a default type of `History<unknown>`).

Suggest using `any` exceptionally.